doc/wg/core/notes/core-notes-2025-02-26.md

```diff
@@ -0,0 +1,109 @@
+
+## PRs
+ - Update nightly (https://github.com/tock/tock/pull/4348)
+    - Brad: Mostly just clippy changes, biggest thing is the register macro recursion limit, which we can just increase as workaround for now
+    - Leon: Rust 2024 edition has been stabilized. Biggest challenge we will have is that static muts are now hard errors. This nightly (and corresponding stable) would let us upgrade to 2024 edition
+    - Brad: Yeah... the static mut issue is hanging over our head still
+    - Leon: Yeah, so now is probably the time to revisit this and figure out a solution
+    - Brad: Agreed, we don't want to fall too far behind, and don't want to be stuck on 2021 edition
+    - Pat: I have a follow-on WIP patch removing yet more semicolons would like to add before merging
+    - Brad: Sounds good!
+
+
+ - Dynamic Process Loading (https://github.com/tock/tock/pull/3941)
+    - Viswajith: Experimenting with writing the app but not writing the header (or header getting written in pieces?)
+    - Viswajith: Currently we guarantee that the first 8 bytes are known
+    - Viswajith: The current TBF threat model is that we only rely on the total_size field in the header and nothing else
+    - Viswajith: Thus ultimately we were considering that header parsing not rely on checksums
+    - Leon: Don't quite follow when this is an issue?
+    - Viswajith: If you have a new app being written to flash, the currentDPL implementation doesn't make assumptions about writing app atomically, nor the order that segments of app is written to flash
+    - Viswajith: If an app is partially through being updated, when current code scans flash, it might look like a padding app but the checkshum will fail (as the contents isn't empty padding)
+    - Brad: This is a question for Johnathan. Currently, we need the total size to find the next app in flash. 
+    - Johnathan: Didn't see any discussion in this issue?
+    - Johnathan: Two issues here: There is integrity/availability. App loading can't break the linked list. The other issue is confidentiality, if, say all the apps are 1K in size, then if an app sets its size to 10K then it gains the ability to read the (presumed secret) app binary
+    - Johnathan: That said, I don't really understand where the issue is in the context of this PR
+    - Brad: The second issue, confidentiality, is easy to handle and is handled
+    - Brad: Right now, the implementation respects what the threat model says it does. One option would be to update the threat model. Other change the loader somehow.
+    - Johnathan: Let me see if I understand right, there is also a checksum on the header, and if that checksum fails the loader doesn't trust it and the list breaks?
+    - Viswajith: Not quite...
+    - Johnathan: Conceptually there should be something file-system like that lets you find where apps are located that are trusted, and then app headers which have to be trusted; the intrusive linked list mixes two trust levels in the same header
```

Comment:
"app headers which have to be trusted" should be "app headers which have to be untrusted"